Cpp/odin-views/app.cpp

```diff
@@ -49,8 +50,14 @@ namespace {
             const auto app_id = paths[0];
             fostlib::json app = odin::app::get_detail(cnx, app_id);
             cnx.commit();
-            if ( app.isnull() )
+            if ( app.isnull() ) {
                 throw fostlib::exceptions::not_implemented(__PRETTY_FUNCTION__, "App not found");
+            }
+
+            if ( fostlib::coerce<fostlib::string>(app["app"]["access_policy"]) != "INVITE_ONLY" ) {
```

Comment:
Not a fan of this hard coded string. Where else does it appear?

Reply:
Only here. I think we can improve it over time. For now, it's Okay.

Reply:
Probably still better to have a defined constant somewhere